awkward/array/jagged.py

```diff
@@ -1350,9 +1362,42 @@ def concatenate(isclassmethod, cls_or_self, arrays):
 
         return cls(starts, stops, content)
 
-    @classmethod
-    def stack(cls, first, *rest):    # each item in first followed by second, etc.
-        raise NotImplementedError
+    @awkward.util.bothmethod
+    def stack(isclassmethod, cls_or_self, arrays):    # each item in first followed by second, etc.
```

Comment:
As in my first comment, I think this should be the `axis=1` case of `concatenate`, for better agreement with Numpy. Also, that means you wouldn't have to copy the cls_or_self handling.